src/Search/Search.Tests/Tests/CustomAnalyzerTests.cs

```diff
@@ -273,7 +273,7 @@ public void CanCreateAllAnalysisComponents()
                         tokenChars: new[] { TokenCharacterKind.Letter }),
                     new NGramTokenizer(SearchTestUtilities.GenerateName(), minGram: 2, maxGram: 4, tokenChars: new[] { TokenCharacterKind.Letter }),
                     new ClassicTokenizer(SearchTestUtilities.GenerateName(), maxTokenLength: 100),
-                    new KeywordTokenizer(SearchTestUtilities.GenerateName(), bufferSize: 100),
```

Comment:
@shahabhijeet This is a tricky case in general because `KeywordTokenizer` is marked with `[Obsolete]`, so using it results in compiler warnings. Also, the new API version (next PR after this one) disallows creating `KeywordAnalyzer` instances, so in order to test this we'd have to use an older version of the SDK to create it first (or use a `DelegatingHandler` to mess with the `api-version` in the request URI). While all of this is possible, it doesn't seem very beneficial for testing classes like `KeywordAnalyzer` that will never change again (since they're deprecated).